New blog post (#1812)
withastro/astro.build: src/content/blog/mux-official-video-partner.mdx

---
title: "Mux: Our Official Video Partner"
authors:
  - fred
description: |
  We are happy to announce Mux as our official video partner, donating $5,000 each month toward the ongoing open-source maintenance and development of Astro.
homepageLink:
  title: Astro ❤️ Mux
  subtitle: Our New Official Video Partner
publishDate: "September 9, 2025"
socialImage: "/src/content/blog/_images/mux-official-video-partner/astro-mux-social.webp"
coverImage: "/src/content/blog/_images/mux-official-video-partner/astro-mux-header.webp"

> lang: "en"

Reviewer: ```suggestion
lang: "en"
related:
  - codetv
```
This might break the build without the other post being published yet, (and this is a case study slug, so hopefully that works here) so do not commit this yet!

Reviewer 2: Shoudl work now

---

import Blockquote from "/src/components/Blockquote.astro"
import BlogContentImage from "/src/components/BlogContentImage.astro"

> I am thrilled to announce [Mux](https://www.mux.com?utm_campaign=21819274-Astro&utm_source=astro) as our new **Official Video Partner**. Mux will be sponsoring Astro for $5,000 each month going forward to support the ongoing open-source maintenance and development of Astro. Plus, we'll be working closely with them on a few new projects (more below).

Reviewer 2: ```suggestion
I am thrilled to announce [Mux](https://www.mux.com?utm_campaign=21819274-Astro&utm_source=astro) as our new **Official Video Partner**. Mux has made a long-term commitment to sponsor Astro, supporting the ongoing open-source maintenance and development of Astro. Plus, we'll be working closely with them on a few new projects (more below).
```

[Mux](https://www.mux.com?utm_campaign=21819274-Astro&utm_source=astro) provides video infrastructure for the web that makes it easy for development teams to ship high-performance, cost-effective video. We see video as a core primitive on the web, so naturally we jumped at the opportunity to partner together to make it even easier for Astro developers to work with video at scale.

<Blockquote>
	<Fragment slot="quote" >
		Astro has always impressed me with their focus and commitment to web performance and developer experience. They do so much well, but the best developer experience is when things just work. I'm excited that our partnership means developers using Astro can add high-performance video without sacrificing the speed and simplicity that drew them to Astro in the first place.
	</Fragment>
	<Fragment slot="caption">
		— [Matt McClure](https://x.com/matt_mcclure), Cofounder and Head of Developer Experience
	</Fragment>
</Blockquote>

## Astro ❤️ Mux

> As part of this partnership, official Astro components for the Mux Player are now available on npm at [@mux/mux-player-astro](https://www.npmjs.com/package/@mux/mux-player-astro). 

Reviewer 2: ```suggestion
This partnership includes two new official Astro components for Mux that you can use today for unparalleled video DX and performance in your Astro projects.

An Astro-native Mux Player component is now available on npm at [@mux/mux-player-astro](https://www.npmjs.com/package/@mux/mux-player-astro). 
```

I think it's probably still worth including "for Mux" or equivalent

> As part of this partnership, official Astro components for the Mux Player are now available on npm at [@mux/mux-player-astro](https://www.npmjs.com/package/@mux/mux-player-astro). 

Reviewer: Agreed!

```astro
---
import { MuxPlayer } from "@mux/mux-player-astro";
---
<MuxPlayer
  playbackId="DS00Spx1CV902MCtPj5WknGlR102V5HFkDe"
  metadata={{ video_title: 'My Video' }}
/>
```

We've worked closely with the Mux team to develop this integration and ensure that it's easy to use and works well with Astro. Check out our [new documentation](https://docs.astro.build/en/guides/media/mux/) to learn more about these new components and how you can use Mux in any Astro project.

If you need an account, you can get started with Mux for free at [mux.com](https://www.mux.com?utm_campaign=21819274-Astro&utm_source=astro). To celebrate our partnership, new Astro users can get an additional $50 credit at signup by using the code `ASTRO`.

> We are so grateful to the entire Mux team for both their financial support and evangelism. The new sponsorship funding will be invested directly back into our open-source community and the long-term maintenance of Astro, benefiting everyone. You can learn more about how we use these funds by reading our [FUNDING.md](https://github.com/withastro/.github/blob/main/FUNDING.md) document on GitHub.

Reviewer: ```suggestion
We are so grateful to the entire Mux team for both their financial support and enthusiasm. The new sponsorship funding will be invested directly into the open-source project to support the long-term maintenance of Astro, benefiting everyone. You can learn more about how we use these funds by reading our [FUNDING.md](https://github.com/withastro/.github/blob/main/FUNDING.md) document on GitHub.
```
"evangalism" really didn't feel like the right word here: enthusiasm was the first thing I could think of but ... collaboration, "(joint) committment to...", "contribution to the Astro ecosystem at large" , "and for choosing Astro as...(their next development partner)"

I think something more along the lines of thanking them for something above fits better.

Reviewer 2: ```suggestion
We are so grateful to the entire Mux team for their support for Astro. The new sponsorship funding will be invested directly into the open-source project to support the long-term maintenance of Astro, benefiting everyone. You can learn more about how we use these funds by reading our [FUNDING.md](https://github.com/withastro/.github/blob/main/FUNDING.md) document on GitHub.
```

Maybe just drop it
